src/prisma-generator.ts

```diff
@@ -28,21 +28,24 @@ export async function generate(options: GeneratorOptions) {
   await fs.mkdir(outputDir, { recursive: true });
   await removeDir(outputDir, true);
 
-  options.generator.config["isGenerateSelect"] = "true";
```

Comment:
I added this line because my previous trials showed me that typescript errors would occur, so keeping it always enabled seemed logical to me.

Reply:
If we keep this line then we can not disable the select flag. Does this not make the `select` flag useless?
Also, I am getting any error. Am I missing something?!

Reply:
I just tested the generated files without select/include, and it seems to be working fine with just "input".